bnbridge/src/components/instructions/instructions.jsx

```diff
@@ -147,6 +173,20 @@ class Instructions extends Component {
     )
   };
 
+  render() {
+    const {
+      classes,
+      swapDir
+    } = this.props;
+
+    return (
+      <Grid container className={ classes.root }>
+          { swapDir === 1 && this.renderBEP2_ERC20() }
+          { swapDir === 2 && this.renderERC20_BEP2() }
```

Comment:
define const or enum for swapDir instead of using 1, 2 hardcoded. makes code readable